### PR TITLE
feat: root_symbol and indent_markers are added

### PR DESCRIPTION
**Pull request by wuelnerdotexe**
Hello again @lambdalisue !! - thanks for the corrections in the [previous pull request](https://github.com/lambdalisue/fern-renderer-nerdfont.vim/pull/19), I have opened this new contribution with all the corrections clean. I hope that this time they meet all the standards of their projects.

Therefore, I recommend that you spend your time to have the necessary standards and guidelines in all Fern-related projects, so that more developers are encouraged to bring more interesting features to the much-loved Fern.

--
Thanks again,
Wuelner.

### CODE REVIEW

File: doc/fern-renderer-nerdfont.txt [hunk 1]
```diff
@@ -52,6 +52,15 @@ VARIABLE				*fern-renderer-nerdfont-variable*
 	symbol.
 	Default: " "
 
+*g:fern#renderer#nerdfont#root_symbol*
+	A |String| used as a symbol of root node.
+	Default: ""
+
+*g:fern#renderer#nerdfont#markers*
```
Comment (lambdalisue) [inline doc/fern-renderer-nerdfont.txt]: Wrong name.

File: autoload/fern/renderer/nerdfont.vim [hunk 2]
```diff
@@ -19,16 +19,54 @@ endfunction
...
+          let levels[key] = 0
+        endif
+      endfor
+
+      let levels[node._renderer_nerdfont_level] = 1
+    endfor
+
+    for i in range(length_nodes)
+      let node = a:nodes[i]
+      let last_marker = i == 0 ? [0] : a:nodes[i - 1]._renderer_nerdfont_marker
```
Comment (lambdalisue) [inline autoload/fern/renderer/nerdfont.vim]: `is#`

File: autoload/fern/renderer/nerdfont.vim [hunk 3]
```diff
@@ -39,24 +77,47 @@ function! s:syntax() abort
...
+  highlight default link FernBranchText    Statement
+  highlight default link FernIndentMarkers NonText
 endfunction
 
 function! s:render_node(node, base, options) abort
   let level = len(a:node.__key) - a:base
   if level is# 0
     let suffix = a:node.label =~# '/$' ? '' : '/'
-    return a:node.label . suffix . '' . a:node.badge
+    let padding = a:options.root_symbol == '' ? '' : a:options.padding
```
Comment (lambdalisue) [inline autoload/fern/renderer/nerdfont.vim]: `==#`

**Review by lambdalisue — CHANGES_REQUESTED** [review]
A bit more please

**Comment by wuelnerdotexe** [comment]
@lambdalisue sorry, I have already corrected them...

**Review by lambdalisue — APPROVED** [review]
LGTM

**Comment by wuelnerdotexe** [comment]
@lambdalisue thank you for your patience !! \0/

**Pull request merged**